### PR TITLE
DEV-1494: Update existing docs to include tooltips where relevant

### PR DESCRIPTION
**Pull request by ShreyaThapa**
FYI, older commits from [DEV-1454](https://github.com/Dwolla/dev-portal/commit/6198422d99b7d96324ac83658cf78ee8aa9cd8cb) also got ported over, which are already in main. Only the following commits need reviewing:

[DEV-1494: Update existing docs to include tooltips where relevant](https://github.com/Dwolla/dev-portal/pull/556/commits/8ea686d9ebab1b391768b8dd5090655303b6d797)

[Add ContenTooltip preset for webhook](https://github.com/Dwolla/dev-portal/pull/556/commits/a10ec318787431e31415786c5c11a3d42017e071)

### CODE REVIEW

File: app/components/tooltip-presets.ts [hunk 1]
```diff
@@ -8,5 +10,47 @@ export default {
...
+      title: "Idempotency key",
+    },
+    "openapi-spec": {
+      content:
+        "A standardized interface description for HTTP/REST APIs, facilitating understanding without source code access. Can be combined with additional tooling, such as generating documentation, a Postman collection, or a Software Development Kit.",
+      title: "Open API Specification",
+    },
+    sandbox: {
+      content:
+        "A simulation of Dwolla's production environment, complete with all API endpoints, but without real banking or user data.",
```
Comment (spencerhunter) [inline app/components/tooltip-presets.ts:41]: Suggested edit:
A simulated environment where you can imitate production/“live” use cases, safely test your API requests and responses in a controlled setting, and mock out low-volume and high volume data.

File: app/components/tooltip-presets.ts [hunk 2]
```diff
@@ -8,5 +10,52 @@ export default {
...
     },
+    "dwolla-connect": {
+      content:
+        "Dwolla Connect leverages your existing banking relationships and corresponding commercial bank accounts to integrate modern payment processing capabilities into your operations.",
+      icon: DwollaConnectIcon,
+      title: "Dwolla Connect",
+    },
+    "external-party": {
+      content:
+        "A type of Dwolla account, either business or personal, that identifies your end-users. An External Party is required in order to initiate a transfer to a customer’s bank account.",
```
Comment (spencerhunter) [inline app/components/tooltip-presets.ts]:
```suggestion
        "A type of end-user, either business or personal, that identifies the owner of the underlying bank account used in an account-to-account payment. An External Party is required in order to initiate a transfer to or from a user's bank account.",
```

File: app/components/tooltip-presets.ts [hunk 3]
```diff
@@ -8,5 +10,52 @@ export default {
...
+      title: "Idempotency key",
+    },
+    "openapi-spec": {
+      content:
+        "A standardized interface description for HTTP/REST APIs, facilitating understanding without source code access. Can be combined with additional tooling, such as generating documentation, a Postman collection, or a Software Development Kit.",
+      title: "Open API Specification",
+    },
+    sandbox: {
+      content:
+        "A simulation of Dwolla's production environment, complete with all API endpoints, but without real banking or user data.",
```
Comment (spencerhunter) [inline app/components/tooltip-presets.ts]:
```suggestion
        "A simulated environment where you can imitate production/“live” use cases, safely test your API requests and responses in a controlled setting, and mock out low-volume and high volume data.",
```

File: app/components/tooltip-presets.ts [hunk 4]
```diff
@@ -8,5 +10,52 @@ export default {
...
+      icon: SandboxIcon,
+      title: "Sandbox",
+    },
+    "secure-exchange": {
+      content:
+        "A secure data-sharing channel between your Dwolla account and Dwolla's ecosystem partners, enabling controlled access, such as connecting a verified bank account, without sensitive data storage.",
```
Comment (spencerhunter) [inline app/components/tooltip-presets.ts]:
```suggestion
        "A secure data exchange that connects Dwolla to trusted Open Finance ecosystem partners, enabling tokenized access to services such as bank account ownership verification.",
```

File: app/components/tooltip-presets.ts [hunk 5]
```diff
@@ -8,5 +10,52 @@ export default {
...
+      title: "Secure Exchange",
+    },
+    "treasury-account": {
+      content:
+        "An identifiable “connection” between Dwolla and a supported Treasury Partner, identified by your ACH Company ID or Gateway Company ID, depending on the banking institution.",
+      title: "Treasury Account",
+    },
+    webhook: {
+      content:
+        "An HTTP request sent to a specified URL for real-time updates and notifications when specific events or data changes occur in your platform.",
```
Comment (spencerhunter) [inline app/components/tooltip-presets.ts]:
```suggestion
        "An HTTP request sent to a specified URL for near real-time updates and notifications when specific events or data changes occur in the Dwolla platform.",
```

File: app/components/tooltip-presets.ts [hunk 6]
```diff
@@ -8,5 +10,52 @@ export default {
...
+      title: "Dwolla Connect",
+    },
+    "external-party": {
+      content:
+        "A type of Dwolla account, either business or personal, that identifies your end-users. An External Party is required in order to initiate a transfer to a customer’s bank account.",
+      title: "External Party",
+    },
+    "funding-source": {
+      content:
+        "A linked bank account that can be used to route funds to a Dwolla Account or External Party, denoted by the use of an account and routing number, most often connected to a checking or savings account.",
```
Comment (spencerhunter) [inline app/components/tooltip-presets.ts]:
```suggestion
        "A linked bank account that allows you to send and receive account-to-account payments with another party. It is identified by a unique account and routing number, and is typically associated with a checking or savings account.",
```

**Review by spencerhunter — COMMENTED** [review]
A few suggested edits on the tooltip text :)

**Comment by ShreyaThapa** [comment]
> A few suggested edits on the tooltip text :)

Awesome! Thanks for the edits. :)

Batched and committed them here -  [2ea7828](https://github.com/Dwolla/dev-portal/pull/556/commits/2ea78285eafd1ba3d0962dc896b08cffc964a421)

**Review by spencerhunter — APPROVED** [review]

**Pull request merged**